CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Vx.x.x (Pre-release)
 
 ## Changes since v3.1.0
+- Improvements to authenticate requests with a JWT bearer token in the `Authorization` header via
```

Comment:
Please start the changelog note with the PR number, link, summary and author before adding the description (see other changelog entries below for the style